README.md

```diff
@@ -59,6 +59,30 @@ packages via the command line.
   * [Fedora package] stable release: `# dnf install freeorion`
   * [ArchLinux package] stable release
 
+## Directories
+* install directory  
+The location of this README.md file, called `<install_dir>` below.  
+* config - game settings  
+    * linux  
+`$XDG_CONFIG_HOME/freeorion` which defaults to `~/.config/freeorion`  
+    * OSX  
+`Username/Library/Application Support/FreeOrion/`  
+    * Windows  
+`C:\Users\Username\AppData\Roaming\FreeOrion`  
```

Comment:
similar: `$APPDATA\FreeOrion` https://github.com/freeorion/freeorion/blob/master/util/Directories.cpp#L374 (and same with data dir)